gitmanager.py

```diff
@@ -306,20 +306,56 @@ def remove_from_blacklist(cls, item, username, blacklist_type="", code_privilege
     @classmethod
     def merge_pull_request(cls, pr_id, comment=""):
         response = requests.get("https://api.github.com/repos/{}/pulls/{}".format(GlobalVars.bot_repo_slug, pr_id))
-        if not response:
+        files = requests.get("https://api.github.com/repos/{}/pulls/{}/files".format(GlobalVars.bot_repo_slug, pr_id))
+        if not response or not files:
             raise ConnectionError("Cannot connect to GitHub API")
         pr_info = response.json()
+        files_info = files.json()
         if pr_info["user"]["login"] != "SmokeDetector":
             raise ValueError("PR #{} is not created by me, so I can't approve it.".format(pr_id))
         if "<!-- METASMOKE-BLACKLIST" not in pr_info["body"]:
             raise ValueError("PR description is malformed. Blame a developer.")
         if pr_info["state"] != "open":
             raise ValueError("PR #{} is not currently open, so I won't merge it.".format(pr_id))
+        string = pr_info["title"]
+        file = files_info[0]["filename"]
```

Comment:
`string` and `file` are not descriptive variable names. Name variables so their purpose is clear without needing to trace code paths.